pages/common/pystun3.md

```diff
@@ -0,0 +1,16 @@
+# pystun3
+
+> Classic STUN client written in python.
+> More information: <https://github.com/talkiq/pystun3>.
+
+- Make a STUN request:
+
+`pystun3`
+
+- Make a STUN request and specify the stun server:
+
+`pystun3 --stun-host {{stun.1und1.de}}`
```

Comment:
What is the different between host and stun host?

Reply:
In what way? In the first example I want to specify the STUN server. I do this with `--stun-host`. In the second example I want to specify the local source port. I do this with `--source-port`. Whereby the second is only conditionally useful if the NAT discovery algorithm of the Classic STUN is used.

Reply:
Ohh, now I see what you mean. I get the following output:
```
```
usage: pystun3 [-h] [-d] [-H STUN_HOST] [-P STUN_PORT] [-i SOURCE_IP]
               [-p SOURCE_PORT] [--version]

optional arguments:
  -h, --help            show this help message and exit
  -d, --debug           Enable debug logging (default: False)
  -H STUN_HOST, --stun-host STUN_HOST
                        STUN host to use (default: None)
  -P STUN_PORT, --stun-port STUN_PORT
                        STUN host port to use (default: 3478)
  -i SOURCE_IP, --source-ip SOURCE_IP
                        network interface for client (default: 0.0.0.0)
  -p SOURCE_PORT, --source-port SOURCE_PORT
                        port to listen on for client (default: 54320)
  --version             show program's version number and exit
```

Reply:
It seems like the documentation is wrong: https://github.com/talkiq/pystun3/blob/master/stun/cli.py#L19

Reply:
> It seems like the documentation is wrong: https://github.com/talkiq/pystun3/blob/master/stun/cli.py#L19

Yep, I was referring to it. Maybe I will check the code instead.